srcpkgs/libexif/template

```diff
@@ -1,16 +1,21 @@
 # Template file for 'libexif'
 pkgname=libexif
-version=0.6.21
-revision=7
+version=0.6.22
+revision=1
+wrksrc="libexif-libexif-0_6_22-release"
 build_style=gnu-configure
 configure_args="ac_cv_path_DOXYGEN=false"
+hostmakedepends="automake libtool glib-devel gettext gettext-devel"
 short_desc="EXIF file library"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Benjamín Albiñana <benalb@gmail.com>"
 license="LGPL-2.1-or-later"
-homepage="http://sourceforge.net/projects/libexif"
-distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.bz2"
-checksum=16cdaeb62eb3e6dfab2435f7d7bccd2f37438d21c5218ec4e58efa9157d4d41a
-patch_args="-Np1"
+homepage="https://github.com/libexif/libexif"
+distfiles="https://github.com/libexif/libexif/archive/libexif-0_6_22-release.tar.gz"
```

Comment:
`version` should be actually used here, please change to `${version//./_}`

Reply:
Done.

Reply:
And remove old line please.

Reply:
fixed,